### PR TITLE
Fix validator info

### LINKED ISSUE #361: Fix default ROI
Should not be `-100%` :D

### LINKED ISSUE #363: Balance/ROI/Status

### PR DESCRIPTION
**Pull request by BeroBurny**
closes #363 
closes #361 
additional refactor on chart data that I forger in the last pr

### CODE REVIEW

**Review by morrigan — APPROVED** [review]
LGTM

File: src/renderer/components/Cards/ValidatorStat.tsx [hunk 1]
```diff
@@ -4,29 +4,37 @@ import {utils} from "ethers";
...
-                {props.value >= 0 ? "+" : ""}
-                {props.value}%
-            </h1>
-            <h5 className={props.value >= 0 ? "plus" : "minus"}>{props.type}</h5>
+            {props.value === undefined ? (
+                <h1>N/A</h1>
+            ) : (
+                <h1 className={props.value >= 0 ? "plus" : "minus"}>
+                    {props.value >= 0 ? "+" : ""}
+                    {props.value}%
```
Comment (mpetrunic) [inline src/renderer/components/Cards/ValidatorStat.tsx:18]: shoudln't this be formatted to be rounded to two decimals?

File: src/renderer/ducks/validator/sagas.ts [hunk 2]
```diff
@@ -62,13 +50,16 @@ function* loadValidatorsSaga(): Generator<
...
                 return {
                     name: keyStore.getName() ?? `Validator - ${index}`,
                     status: undefined,
                     publicKey: keyStore.getPublicKey(),
-                    network: auth.getValidatorNetwork(keyStore.getPublicKey()),
+                    network,
+                    balance,
                     keystore: keyStore,
-                    isRunning: undefined,
+                    isRunning: false,
```
Comment (mpetrunic) [inline src/renderer/ducks/validator/sagas.ts:62]: Aren't you suppose to take this value depending on if lodestar-validator is running or not?

Comment (BeroBurny) [reply src/renderer/ducks/validator/sagas.ts:62]: Definitely loadstar validator is not running, so seeing no point to keep status undefined if we are sure he is not started and another case `undefined` is a falsy value at the end is same result

File: src/renderer/components/Cards/ValidatorStat.tsx (hunk 1 above)
Comment (BeroBurny) [reply src/renderer/components/Cards/ValidatorStat.tsx:18]: dunno just extended to display `N/A` if there is no value

Comment (mpetrunic) [reply src/renderer/components/Cards/ValidatorStat.tsx:18]: let's format it to be on safe side

File: src/renderer/ducks/validator/sagas.ts (hunk 2 above)
Comment (mpetrunic) [reply src/renderer/ducks/validator/sagas.ts:62]: do we change it somewhere to true?

Comment (BeroBurny) [reply src/renderer/ducks/validator/sagas.ts:62]: after starting a validator

File: src/renderer/components/Cards/ValidatorStat.tsx (hunk 1 above)
Comment (BeroBurny) [reply src/renderer/components/Cards/ValidatorStat.tsx:18]: i check now and yeah its formatted to 2 decimals at `calculateROI` function
https://github.com/NodeFactoryIo/ChainGuardian/blob/78743d0b0d429ea60e5c2745d081274762cb9c67/src/renderer/services/utils/math.ts#L16

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**